GI03FLMA/GI03FLMA_D00009_ingest.csv

```diff
@@ -0,0 +1,20 @@
+#,/omc_data/whoi/OMC/GI03FLMA/D00009/*ctdmo*.dat,GI03FLMA-RIM01-02-CTDMOG041,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes.No acoustic comms with FLMA modem
+#,/omc_data/whoi/OMC/GI03FLMA/D00009/*ctdmo*.dat,GI03FLMA-RIM01-02-CTDMOG042,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes.No acoustic comms with FLMA modem
+#,/omc_data/whoi/OMC/GI03FLMA/D00009/*ctdmo*.dat,GI03FLMA-RIM01-02-CTDMOG043,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes.No acoustic comms with FLMA modem
+#,/omc_data/whoi/OMC/GI03FLMA/D00009/*ctdmo*.dat,GI03FLMA-RIM01-02-CTDMOG044,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes.No acoustic comms with FLMA modem
+#,/omc_data/whoi/OMC/GI03FLMA/D00009/*ctdmo*.dat,GI03FLMA-RIM01-02-CTDMOG045,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes.No acoustic comms with FLMA modem
+#,/omc_data/whoi/OMC/GI03FLMA/D00009/*ctdmo*.dat,GI03FLMA-RIM01-02-CTDMOG046,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes.No acoustic comms with FLMA modem
+#,/omc_data/whoi/OMC/GI03FLMA/D00009/*ctdmo*.dat,GI03FLMA-RIM01-02-CTDMOG047,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes.No acoustic comms with FLMA modem
+#,/omc_data/whoi/OMC/GI03FLMA/D00009/*ctdmo*.dat,GI03FLMA-RIM01-02-CTDMOG048,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes.No acoustic comms with FLMA modem
+#,/omc_data/whoi/OMC/GI03FLMA/D00009/*ctdmo*.dat,GI03FLMA-RIM01-02-CTDMOH049,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes.No acoustic comms with FLMA modem
+#,/omc_data/whoi/OMC/GI03FLMA/D00009/*ctdmo*.dat,GI03FLMA-RIM01-02-CTDMOH050,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes.No acoustic comms with FLMA modem
+#,/omc_data/whoi/OMC/GI03FLMA/D00009/*ctdmo*.dat,GI03FLMA-RIM01-02-CTDMOH051,telemetered,Not Expected,Data assigned to refdes after ingestion with wildcard refdes.No acoustic comms with FLMA modem
+#mi.dataset.driver.sio_eng.sio.sio_eng_sio_telemetered_driver,/omc_data/whoi/OMC/GI03FLMA/D00009/*status_1488401.dat,GI03FLMA-RIS01-00-SIOENG000,telemetered,Not Expected,No acoustic comms with FLMA modem
+#mi.dataset.driver.dosta_abcdjm.sio.dosta_abcdjm_sio_telemetered_driver,/omc_data/whoi/OMC/GI03FLMA/D00009/*dosta*.dat,GI03FLMA-RIS01-03-DOSTAD000,telemetered,Not Expected,No acoustic comms with FLMA modem
+#mi.dataset.driver.phsen_abcdef.sio.phsen_abcdef_sio_telemetered_driver,/omc_data/whoi/OMC/GI03FLMA/D00009/*phsen*.dat,GI03FLMA-RIS01-04-PHSENF000,telemetered,Not Expected,No acoustic comms with FLMA modem
+#mi.dataset.driver.flort_dj.sio.flort_dj_sio_telemetered_driver,/omc_data/whoi/OMC/GI03FLMA/D00009/*flort*.dat,GI03FLMA-RIS01-05-FLORTD000,telemetered,Not Expected,No acoustic comms with FLMA modem
```

Comment:
Looks good.  I believe there  are still some troubleshooting efforts going on with this.